src/command_modules/azure-cli-appservice/HISTORY.rst

```diff
@@ -2,6 +2,9 @@
 
 Release History
 ===============
+* functionapp: deprecate `az functionapp devops-build` command. Rename it to `az functionapp devops-pipeline`.
+* functionapp: add examples for command parameters when using `az functionapp devops-pipeline`.
```

Comment:
Since you have to resolve conflicts in HISTORY.rst anyways, this second entry isn't required. Help text changes are explicitly called out as not needing a history entry :)